openstack/octavia/templates/octavia-api-service.yaml

```diff
@@ -3,7 +3,7 @@ kind: Service
 metadata:
   name: octavia-api
   annotations:
-  {{- include "utils.topology.service_topology_mode" . | indent 2 }}
+    {{- include "utils.topology.service_topology_mode" . | indent 2 }}
```

Comment:
Looks like you missed adding the linkerd annotation here.

Reply:
Fixed, thanks !